syncmaster/worker/handlers/file/s3.py

```diff
@@ -29,3 +30,20 @@ def connect(self, spark: SparkSession):
             extra=self.connection_dto.additional_params,
             spark=spark,
         ).check()
+
+    def read(self) -> DataFrame:
+        from pyspark.sql.types import StructType
+
+        options = {}
+        if self.transfer_dto.file_format.__class__.__name__ == "Excel":
+            options = {"inferSchema": True}
```

Comment:
Actually, I'm not sure if `inferSchema` is necessary, as Spark DataFrame seems to interpret Excel column types correctly anyway

Reply:
I also expected such behavior, but the columns still become strings if only `inferSchema` is passed without `df_schema`. On the other hand, `df_schema` explicitly defines the data types, which is why I raised this question. We can still pass `inferSchema=True` and observe where it might be applicable

Reply:
If I remember correctly, we currently don't fill up `df_schema` anywhere, do we?

Reply:
We use it in tests at `init_df` fixture, tried there